src/cert/ssl_cert_provider.cpp

```diff
@@ -117,6 +118,21 @@ std::string cn_name_from(const std::string& server_name)
     return server_name;
 }
 
+void set_san_name(X509* c, std::string server_name)
```

Comment:
```suggestion
void set_san_name(X509* c, const std::string& server_name)
```
We prefer const reference for this.

Reply:
Resolved in latest commit :pray: :rocket: